### DISCUSSION TITLE
Error while installing eksctl.

### DISCUSSION THREAD

**Discussion by doncake** [Homebrew/discussions#3998] [Getting started]
Hi there. I know that the home-brew team and apple don't support the latest versions of Mac OS like me - High Sierra. But may somebody knows how to solve next problem, appearing while installing eksctl by means of command $ brew install weaveworks/tap/eksctl. Thanks in advance.

$ brew install weaveworks/tap/eksctl
Warning: You are using macOS 10.13.
We (and Apple) do not provide support for this old version.
It is expected behaviour that some formulae will fail to build in this old version.
It is expected behaviour that Homebrew will be buggy and slow.
Do not create any issues about this on Homebrew's GitHub repositories.
Do not create any issues even if you think this message is unrelated.
Any opened issues will be immediately closed without response.
Do not ask for help from MacHomebrew on Twitter.
You may ask for help in Homebrew's discussions but are unlikely to receive a response.
Try to figure out the problem yourself and submit a fix as a pull request.
We will review it but may or may not accept it.

==> Downloading https://ftp.gnu.org/gnu/bash/bash-5.2-patches/bash52-001
Already downloaded: /Users/la/Library/Caches/Homebrew/downloads/c3a3624e2f2aba97ca0150fb4d12b89557fa01f173f23a0e9ac0a83a9cedaae5--bash52-001
==> Downloading https://ftp.gnu.org/gnu/bash/bash-5.2-patches/bash52-002
Already downloaded: /Users/la/Library/Caches/Homebrew/downloads/0c69a6e0728e2a524fbe8964a0022d5329da009f861f64bf668d75fcb1455611--bash52-002
==> Downloading https://ftp.gnu.org/gnu/bash/bash-5.2-patches/bash52-003
Already downloaded: /Users/la/Library/Caches/Homebrew/downloads/9b49e7f6e3b5e9a84ce1fad0d6e0904032977f2667aeb6ff32a6eb894747659f--bash52-003
==> Downloading https://ftp.gnu.org/gnu/bash/bash-5.2-patches/bash52-004
Already downloaded: /Users/la/Library/Caches/Homebrew/downloads/405aec744f0bfae0f81476c5ce567a39b7c7d589b87325493591be4847cd2c63--bash52-004
==> Downloading https://ftp.gnu.org/gnu/bash/bash-5.2-patches/bash52-005
Already downloaded: /Users/la/Library/Caches/Homebrew/downloads/14cf3a8bc1d14a21aa5191be71bd69eb24e383e3b48a82224ecedc8150d73719--bash52-005
==> Downloading https://ftp.gnu.org/gnu/bash/bash-5.2-patches/bash52-006
Already downloaded: /Users/la/Library/Caches/Homebrew/downloads/92879b138e46d1b25208d04a83941d2f2e52ead4a9bea7ed2b662099a94053fe--bash52-006
==> Downloading https://ftp.gnu.org/gnu/bash/bash-5.2-patches/bash52-007
Already downloaded: /Users/la/Library/Caches/Homebrew/downloads/8ee2285c98c63f8f108211f70d7bd91a217ecf9bebbcaa3facbd25e9ad52ed5f--bash52-007
==> Downloading https://ftp.gnu.org/gnu/bash/bash-5.2-patches/bash52-008
Already downloaded: /Users/la/Library/Caches/Homebrew/downloads/2dc4950d9c144bc12402bc3d4974d33942c04c36a8db052fb9a3b50c49304617--bash52-008
==> Downloading https://ftp.gnu.org/gnu/bash/bash-5.2-patches/bash52-009
Already downloaded: /Users/la/Library/Caches/Homebrew/downloads/ed7526ab1d9e2f214d46e92c5e474a7db3e6cdff0dadf154a4e6372bac5902cd--bash52-009
==> Downloading https://ftp.gnu.org/gnu/bash/bash-5.2-patches/bash52-010
Already downloaded: /Users/la/Library/Caches/Homebrew/downloads/2176fc50962a713af36faed54a78731afa0bbe338c3fea16f1d5559c29f7e3d8--bash52-010
==> Downloading https://ftp.gnu.org/gnu/bash/bash-5.2-patches/bash52-011
Already downloaded: /Users/la/Library/Caches/Homebrew/downloads/2b2691a8a09a33a64b13828bccadc6b789f5f82d3b4e2b736b89d752e1491db0--bash52-011
==> Downloading https://ftp.gnu.org/gnu/bash/bash-5.2-patches/bash52-012
Already downloaded: /Users/la/Library/Caches/Homebrew/downloads/650a71260c02cfbd9304a72577a5e88ad560aa263a0d24b44f6d8b9f1648d7bb--bash52-012
==> Downloading https://ftp.gnu.org/gnu/bash/bash-5.2.tar.gz
Already downloaded: /Users/la/Library/Caches/Homebrew/downloads/f101067617209f4835f339def33e3a3884b6e2344bef4bc8855610ad06d73bd4--bash-5.2.tar.gz
==> Downloading https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8
Already downloaded: /Users/la/Library/Caches/Homebrew/downloads/195208ba31cadc265ea97bb8aad4e78caab1200daf4d855e6c839b5f987f92db--configure-big_sur.diff
==> Downloading https://gmplib.org/repo/gmp/raw-rev/5f32dbc41afc
Already downloaded: /Users/la/Library/Caches/Homebrew/downloads/b45d6f949d28daf5403e369b6d53935e403ffbeff7f5c035dc88650a0bd76e51--5f32dbc41afc
==> Downloading https://gmplib.org/download/gmp/gmp-6.2.1.tar.xz
Already downloaded: /Users/la/Library/Caches/Homebrew/downloads/ade51fddb351cd85e1645e43543cff053bc2f667ec8a7eb1c7554ac301a30d4b--gmp-6.2.1.tar.xz
==> Downloading https://ftp.gnu.org/gnu/coreutils/coreutils-9.1.tar.xz
Already downloaded: /Users/la/Library/Caches/Homebrew/downloads/6d4074aa69f40ea34f39f181f19a84eeff526ae7a388cb05bda87448d8628061--coreutils-9.1.tar.xz
==> Downloading https://storage.googleapis.com/golang/go1.16.darwin-amd64.tar.gz
Already downloaded: /Users/la/Library/Caches/Homebrew/downloads/c50278859eacb3397c41a3baef31fd7b40fa2c5f08912812051bb36f777c3b7c--go1.16.darwin-amd64.tar.gz
==> Downloading https://go.dev/dl/go1.19.3.src.tar.gz
Already downloaded: /Users/la/Library/Caches/Homebrew/downloads/e95d12588643f7a9c6087dfcb33aaf914477e45c7d50c792677a1ee4fe8ef194--go1.19.3.src.tar.gz
==> Cloning https://github.com/kubernetes/kubernetes.git
Updating /Users/la/Library/Caches/Homebrew/kubernetes-cli--git
==> Checking out tag v1.25.4
HEAD is now at 872a965c6c6 Release commit for Kubernetes v1.25.4
HEAD is now at 872a965c6c6 Release commit for Kubernetes v1.25.4
==> Cloning https://github.com/kubernetes-sigs/aws-iam-authenticator.git
Updating /Users/la/Library/Caches/Homebrew/aws-iam-authenticator--git
==> Checking out tag v0.5.11
HEAD is now at 8e851c24 Update to version v0.5.11 (#515)
HEAD is now at 8e851c24 Update to version v0.5.11 (#515)
Error: v0.5.11 tag should be 918bfc9fb357618410d0a6d31b53c757840bd69e
but is actually 8e851c244de20424354d0f911150acac498e6b35

**Comment by chenrui333** [comment]
> Error: v0.5.11 tag should be 918bfc9fb357618410d0a6d31b53c757840bd69e
> but is actually 8e851c244de20424354d0f911150acac498e6b35

This would be the fix, https://github.com/Homebrew/homebrew-core/pull/117023, thanks for raising it.

**Comment by chenrui333** [comment]
@doncake this would be [the fix](https://github.com/Homebrew/homebrew-core/pull/117166)
